libs/elements/layout/images/src/lib/components/elewa-image-container/elewa-image-container.component.html

```diff
@@ -0,0 +1,63 @@
+<div class="image-container" *ngFor="let item of images">
+  <div *ngIf="item.visualisation === 0">
+    <h3 class="title">{{ item.title }}</h3>
+    <div class="contain">
+      <img
+        [ngClass]="{
+        pill: item.visualisation === 0,
+      }"
+        [style.width]="item.maxWidth"
+        src="{{ item.imageSrc }}"
+        alt=""
+      />
+    </div>
+  </div>
+
+  <div *ngIf="item.visualisation === 1">
+    <h3>{{ item.title }}</h3>
+    <div class="stack">
+      <div class="image">
+        <img src="{{ item.imageSrc }}" alt="" />
+      </div>
+      <div class="triangle"></div>
+      <div class="triangle mid-left"></div>
+      <div class="frame"></div>
+      <div class="stacked-container">
+        <div class="stacked stacked1"></div>
+        <div class="stacked stacked2"></div>
+      </div>
+      <div class="stacked-container1">
+        <div class="stacked stacked3"></div>
+        <div class="stacked stacked4"></div>
+      </div>
+    </div>
+  </div>
```

Comment:
This should happen dynamically. Use the ngClass directive to set the class on the image depending on it's imageConfig

Reply:
There's no need to create a div for every type of image Visualisation. use ngfor to loop and ngclass to set the class